index.html

```diff
@@ -5165,10 +5165,6 @@ <h2 id="AudioWorkerParamDescriptor">
             readonly attribute AudioParam orientationX
           </dt>
           <dd>
-            The <code>orientationX, orientationY, orientationZ</code>
```

Comment:
We should probably move this into a different section, just to make it clear what these are.  But that's a different PR.